libraries/ESP32/examples/Camera/CameraWebServer/app_httpd.cpp

```diff
@@ -1304,3 +1315,14 @@ void startCameraServer()
         httpd_register_uri_handler(stream_httpd, &stream_uri);
     }
 }
+
+void ledc_setup()
```

Comment:
`ledc_setup()` could have as parameters all the information declared in the `#define`, making it more generic and clear.

Reply:
> `ledc_setup()` could have as parameters all the information declared in the `#define`, making it more generic and clear.

This is possible of course, but requre a lot of changes and still requre a defines since some boards can have different GPIO pins for led or not have a led at all. I can prepare such changes if you ready to review it, but first of all please look to the comment above where discussing about example structure. Can I append new  header file for defines or not? I will move forvard depending of answer.

Reply:
Yes, ... maybe this example should be refactored for good.
Let's keep it simple for now. Keep it as you proposed.

Reply:
This example is a copy of another example we have for ESP-IDF and turned into Arduino sketch. Structure allows changes if they make sense :)